src/test/java/com/google/sps/servlets/NewRequestServletTest.java

```diff
@@ -95,23 +124,33 @@ public void testStartTimeAfterEndTime() throws IOException, ServletException {
     verify(response).sendError(HttpServletResponse.SC_BAD_REQUEST, "Please enter a valid time!");
   }
   /**
-   * Tests that when correct data is sent to the servlet, no errors are sent.
-   * This test will be added when Firebase Credentials will be set
+   * Tests that when correct data is sent to the servlet, no BAD_REQUEST errors are sent. 
    */
-//   @Test
-//   public void testValidData() throws IOException, ServletException {
-//     when(request.getParameter("delivery-date")).thenReturn("2020-09-26");
-//     when(request.getParameter("start-time")).thenReturn("05:15");
-//     when(request.getParameter("end-time")).thenReturn("15:10");
-//     when(request.getParameter("max-stops")).thenReturn("1");
-//     when(request.getParameter("timezone-offset")).thenReturn("180");
-
-//     NewRequestServlet requestServlet = new NewRequestServlet();
-//     requestServlet.doPost(request, response);
-//     // the date is valid, thus "Please enter a valid date!" shouldn't be sent
-//     verify(response, times(0)).sendError(HttpServletResponse.SC_BAD_REQUEST, "Please enter a valid date!");
-//     verify(response, times(0)).sendError(HttpServletResponse.SC_BAD_REQUEST, "Please enter a valid time!");
-//     verify(response, times(0)).sendError(HttpServletResponse.SC_BAD_REQUEST, "Please enter between 1 and 25 stops!");
-//     //TODO[ak47na]: check that the delivery request is added to Mock datastore
-//   }
+  @Test
+  public void testValidData() throws IOException, ServletException, FirebaseAuthException {
```

Comment:
This looks ok for now.

For future reference I would advise to get everything out of the servlet code in order to make testing easier. This avoids having to mock a lot of the firebase stuff in the future:
https://github.com/googleinterns/step235-2020/pull/1/files#r481166522